contracts/config/Config.sol

```diff
@@ -679,9 +676,11 @@ contract Config is ReentrancyGuard, RoleModule, BasicMulticall {
             baseKey == Keys.POSITION_FEE_RECEIVER_FACTOR ||
             baseKey == Keys.SWAP_FEE_RECEIVER_FACTOR ||
             baseKey == Keys.BORROWING_FEE_RECEIVER_FACTOR ||
+            baseKey == Keys.LIQUIDATION_FEE_RECEIVER_FACTOR ||
```

Comment:
seems not needed since LIQUIDATION_FEE_FACTOR is validated above

Reply:
i think it still makes sense to validate both LIQUIDATION_FEE_RECEIVER_FACTOR and LIQUIDATION_FEE_FACTOR?